tests/testthat/test-bulk_methods.R

```diff
@@ -851,8 +851,13 @@ test_that("differential trancript abundance - random effects",{
 
 test_that("test prefix",{
 
-	library(DESeq2)
   library(stringr)
+  
+  if (find.package("DESeq2", quiet = TRUE) %>% length %>% equals(0)) {
+    if (!requireNamespace("BiocManager", quietly = TRUE))
+      install.packages("BiocManager", repos = "https://cloud.r-project.org")
+    BiocManager::install("DESeq2", ask = FALSE)
+  }
```

Comment:
I will add this to my PR, see we keep 

- drop DESeq2 dependency, and
- resolve conflicts

independent.